spec/sections_spec.rb

```diff
@@ -19,11 +19,11 @@ class APage < SitePrism::Page
     end
 
     it 'should create matching object method' do
-      expect(page).to respond_to(:single_section)
+      expect(subject).to respond_to(:single_section)
```

Comment:
Nagging you about this one last time....If you read my notes above you'll already know that I am strongly in favor of using named subjects.  (The RSpec relish docs seem to point in that direction as well.)  Please consider changing this.

Reply:
Furthermore, each of this it blocks can be easily made more succinct:

```ruby
it { is_exected.to respond_to(:single_section) }
it { is_expected.to respond_to(:has_single_section?) }
it { is_expected.to respond_to(:plural_sections) }
it { is_expected.to respond_to(:has_plural_sections?) }
```

Reply:
Now if you're using these succinct it blocks, that probably reduces the utility of a named subject. But then I'd argue that your spec would be short enough that an unnamed subject would no longer be problematic (i.e. it wouldn't be hard to know what the subject is for each assertion).